assets/app/views/_overview-deployment.html

```diff
@@ -49,7 +49,7 @@ <h3 class="h4">
           <span class="deployment-trigger" ng-repeat="cause in rc.causes">
             <span ng-switch="cause.type">
               <span ng-switch-when="ImageChange">
-                a new image <span ng-if="cause.imageTrigger.from">for {{cause.imageTrigger.from | imageObjectRef : null : true}}</span>
+                a new image <span ng-if="cause.imageTrigger.from" style="white-space: nowrap;">for {{cause.imageTrigger.from | imageObjectRef : null : true}}</span>
```

Comment:
We should add the `white-space` style just around the angular expression so the line break is after the "for."

```
<span ng-if="cause.imageTrigger.from">for <span style="white-space: nowrap;">{{cause.imageTrigger.from | imageObjectRef : null : true}}</span></span>
```

(Sorry that was not what I asked before. My fault.)

Even better: can you try to use the Bootstrap `text-nowrap` class?

```
<span ng-if="cause.imageTrigger.from">for <span class="text-nowrap">{{cause.imageTrigger.from | imageObjectRef : null : true}}</span></span>
```